sopel/bot.py

```diff
@@ -1193,6 +1177,11 @@ def search_url_callbacks(self, url):
 
         .. versionadded:: 7.0
 
+        .. versionchanged:: 8.0
+
+            Searches for registered callbacks in an internal property instead
```

Comment:
Wait a second... shouldn't this be deprecated and removed in Sopel 9, like the other methods?

Reply:
Probably. Must have been an oversight in #2049.